tools/llvm-spirv/llvm-spirv.cpp

```diff
@@ -153,6 +153,11 @@ static cl::opt<bool>
     SPIRVToolsDis("spirv-tools-dis", cl::init(false),
                   cl::desc("Emit textual assembly using SPIRV-Tools"));
 
+static cl::opt<bool> EmitOpaquePointers(
+    "emit-opaque-pointers", cl::init(false),
```

Comment:
I guess this flag might have to be removed soon again, when LLVM drops non-opaque pointer support.  So perhaps it makes sense to make it `cl::Hidden`?